<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>binaryDialog</class>
 <widget class="QDialog" name="binaryDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Binarizar Imagen</string>
  </property>
  <widget class="QScrollArea" name="binaryArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="binaryContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>379</height>
     </rect>
    </property>
    <widget class="QLabel" name="binaryLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>380</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSlider" name="thresholdSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>440</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>440</y>
     <width>95</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="thresholdLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>443</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Umbral:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="thresholdBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>441</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="suffix">
    <string/>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aplicar Isodata</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
